src/utils/bundled-editor.jsx

```diff
@@ -58,18 +58,25 @@ function loadPluginsIfEnabled() {
 	const { plugins } = getGBKit();
 
 	if ( plugins ) {
-		return loadEditorAssets().then( ( { allowedBlockTypes } ) =>
-			import( './blocks' ).then( ( { unregisterDisallowedBlocks } ) =>
-				unregisterDisallowedBlocks( allowedBlockTypes )
```

Comment:
Rather than inlining this here, we now pass along the `allowedBlockTypes` value to the editor initialization, which has existing logic for this.